src/hooks/useHotspots.ts

```diff
@@ -33,6 +149,140 @@ const useHotspots = (): {
     dispatch(fetchCollectables({ account: currentAccount, cluster }))
   }, [cluster, collectables.loading, currentAccount, dispatch, l1Network])
 
+  const fetchingMore = useMemo(() => {
+    if (!currentAccount?.solanaAddress) return false
+
+    return collectables[currentAccount?.solanaAddress].fetchingMore
+  }, [collectables, currentAccount])
+
+  const fetchMore = useCallback(() => {
+    if (
+      !currentAccount?.solanaAddress ||
+      l1Network !== 'solana' ||
+      collectables.loading
+    ) {
+      return
+    }
+    dispatch(
+      fetchMoreCollectables({
+        account: currentAccount,
+        cluster,
+        oldestCollectable: oldestCollectableId,
+      }),
+    )
+  }, [
+    cluster,
+    collectables.loading,
+    currentAccount,
+    dispatch,
+    l1Network,
+    oldestCollectableId,
+  ])
+
+  const createHotspot = useCallback(async () => {
+    if (!currentAccount || !anchorProvider || !currentAccount.solanaAddress)
+      return
+
+    const secureStorage = await getKeypair(currentAccount.address)
+    if (!secureStorage) return
+
+    const owner = new Keypair(secureStorage.keypair)
+    const onboardingKey = random(10)
+    const gateway = await Keypair.makeRandom()
+    const maker = Address.fromB58(
+      '13AjXWhBNWdxq63dSQmPvRx3uQtaa3pMu5wXB1bPUZTTYEiwpgC',
+    )
+
+    await axios.post(
+      'https://onboarding.oracle.test-helium.com/api/v2/hotspots',
```

Comment:
Do we want to move this base URL to env vars?

Reply:
Done